src/test/java/net/snowflake/client/RunningNotOnGithubActionsMac.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2012-2019 Snowflake Computing Inc. All right reserved.
+ */
+package net.snowflake.client;
+
+import net.snowflake.client.core.Constants;
+
+public class RunningNotOnGithubActionsMac implements ConditionalIgnoreRule.IgnoreCondition {
+  public boolean isSatisfied() {
```

Comment:
Is there some `@Override` missing?

Reply:
it's not added to any of implementations, but I can add it in all places